zerver/lib/test_classes.py

```diff
@@ -319,29 +312,39 @@ def client_patch(
         self,
         url: str,
         info: Dict[str, Any] = {},
+        skip_user_agent: bool = False,
+        follow: bool = False,
+        secure: bool = False,
         intentionally_undocumented: bool = False,
-        **kwargs: ClientArg,
+        **headers: str,
     ) -> "TestHttpResponse":
         """
         We need to urlencode, since Django's function won't do it for us.
         """
         encoded = urllib.parse.urlencode(info)
         django_client = self.client  # see WRAPPER_COMMENT
-        self.set_http_headers(kwargs)
-        result = django_client.patch(url, encoded, **kwargs)
+        self.set_http_headers(headers, skip_user_agent)
+        result = django_client.patch(url, encoded, follow=follow, secure=secure, **headers)
         self.validate_api_response_openapi(
             url,
             "patch",
             result,
             info,
-            kwargs,
+            headers,
             intentionally_undocumented=intentionally_undocumented,
         )
         return result
 
     @instrument_url
     def client_patch_multipart(
-        self, url: str, info: Dict[str, Any] = {}, **kwargs: ClientArg
+        self,
+        url: str,
+        info: Dict[str, Any] = {},
```

Comment:
Do we have plans to get rid of the `Any` here and in the other similar methods? (out of scope of this PR I think, just curious)

Reply:
Not yet, but it's something that I might also work on.